### DISCUSSION TITLE
A question about merging two GitHub accounts

### DISCUSSION THREAD

**Discussion by st-den** [community/community#23140] [New to GitHub]
<p>Hey there!</p>
<p>I have two accounts: one created in 2013, and the second - current one - created in 2019.</p>
<p>The first account has only one 2015 repo, this one contains all my repos, PRs and issues created during the last few years.</p>
<p>Is there a way I could transfer either the account creation date to this account, or all my work and comments to the first account? Or if that is not possible, will my profile activity feed extend to 2015 if I transfer that one repo I mentioned to this account?</p>
<p>I don’t think the documentation covers my case. Thank you for your help!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave:</p>
<p>We can’t amend a creation date - if you keep the account created in 2019, that will always show as the creation date.</p>
<p>You can transfer your repositories from one account to another:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/merging-multiple-user-accounts" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/merging-multiple-user-accounts" target="_blank" rel="noopener">Merging multiple user accounts - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/merging-multiple-user-accounts" target="_blank" rel="noopener">//docs.github.com/en/github/setting-up-and-managing-your-github-user-account/merging-multiple-user-accounts</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>For commits to be correctly attributed to your new account, you will need to add the email addresses you used for commits in the previous account to the keeper account.</p>
<p>If some of the commits are going to pre-date the creation date on a GitHub account, you might have to <a href="https://support.github.com/contact">open a ticket</a> when you have done all your transferring and ask for your contributions graph to be rebuilt.</p>
<p>However even when that is done, you won’t be able to navigate to any contributions that predate the account creation. You would have to explicitly manipulate the URL to see any period of time that predates account creation, like this:</p>
<pre><code> https://github.com/USERNAME?tab=overview&amp;from=2013-12-01&amp;to=2013-12-31
</code></pre>
<p>I’m sorry this isn’t more straightforward, but I hope this answered your question.</p>

**Reply by jflaga** [reply]
Hi @canuckjacq , is there any plans of putting navigation links to contributions that predate the account creation?
